piprot/piprot.py

```diff
@@ -364,90 +390,33 @@ def main(
-    )
-    cli_parser.add_argument(
-        '-v', '--verbose', action='store_true',
-        help='verbosity, can be supplied more than once '
-             '(enabled by default, use --quiet to disable)'
-    )
-    cli_parser.add_argument('-l', '--latest', action='store_true',
-                            help='print the lastest available version for out '
-                                 'of date requirements')
-    cli_parser.add_argument('-x', '--verbatim', action='store_true',
-                            help='output the full requirements file, with '
-                                 'added comments with potential updates')
-    cli_parser.add_argument('-q', '--quiet', action='store_true',
-                            help='be a little less verbose with the output '
-                                 '(<0.3 behaviour)')
-    cli_parser.add_argument('-o', '--outdated', action='store_true',
-                            help='only list outdated requirements')
-
-    cli_parser.add_argument('-g', '--github',
-                            help='Test the requirements from a GitHub repo. '
-                                 'Requires that a `requirements.txt` file '
-                                 'exists in the root of the repository.')
-
-    cli_parser.add_argument(
-        '-b', '--branch',
-        help='The branch to test requirements from, used with '
-             'the Github URL support.')
-
-    cli_parser.add_argument(
-        '-t', '--token',
-        help='Github personal access token to be used with '
-             'the Github URL support.')
-
-    cli_parser.add_argument(
-        '-p', '--path',
-        help='Path to requirements file in remote repository.')
-
-    cli_parser.add_argument(
-        '-d', '--delay',
-        help='Delay before an outdated package triggers an error.'
-             '(in days, default to 1).')
-
-    cli_parser.add_argument('-u', '--url',
-                            help='URL to requirements file.')
-
-    # if there is a requirements.txt file, use it by default. Otherwise print
-    # usage if there are no arguments.
-    nargs = '+'
-
-    if (
-        '--github' in sys.argv
-        or '-g' in sys.argv
-        or '-u' in sys.argv
-        or '--url' in sys.argv
-    ):
-        nargs = "*"
-
-    default = None
-    if os.path.isfile('requirements.txt'):
-        nargs = "*"
-        default = [open('requirements.txt')]
-
-    cli_parser.add_argument('file', nargs=nargs, type=argparse.FileType(),
-                            default=default, help='requirements file(s), use '
-                                                  '`-` for stdin')
-
-    cli_args = cli_parser.parse_args()
-
-    if len(cli_args.file) > 1 and cli_args.verbatim:
-        sys.exit('--verbatim only allowed for single requirements files')
+    options = docopt(__doc__, version=__version__)
+
+    if options['<file>'] and options['--verbatim']:
+        sys.exit('--verbatim only allowed for single requirements files.')
```

Comment:
There is an extra dot compared to the original.

Reply:
Fixed.